fe/fe-core/src/main/java/com/starrocks/clone/TabletScheduler.java

```diff
@@ -393,6 +410,7 @@ private synchronized void adjustPriorities() {
     private void schedulePendingTablets() {
         long start = System.currentTimeMillis();
         List<TabletSchedCtx> currentBatch = getNextTabletCtxBatch();
+        debugLogPendingTabletsStats();
```

Comment:
should check if LOG.isDebugLevel first before call this function?

Reply:
yes, should check here